tests/bazel_tests.bzl

```diff
@@ -44,6 +44,7 @@ mkdir -p {work_dir}
 mkdir -p {cache_dir}
 cp -f {workspace} {work_dir}/WORKSPACE
 cp -f {build} {work_dir}/BUILD.bazel
+cp -f {extra_files} {work_dir}/
```

Comment:
Wrap this in an `if` in case there are no files. Currently, this prints an error (but proceeds).

Reply:
Done.